RELEASE_SCHEDULE.md

```diff
@@ -9,7 +14,72 @@ You can determine what is in a given build using the [releases page](https://git
 
 Announcements regarding releases will be made to the #headerbidding-dev channel in subredditadops.slack.com.
 
-# FAQs
+## Release Process
+
+1. Make Sure all browserstack tests are passing. Check the last travis build for master branch. If failing please fix the failing tests or skip the test and create issue if it takes too long to fix the failed unit test.
+
+   #### How to run tests in browserstack
+
+   Set the environment variables. You may want to add these to your `~/.bashrc` for convenience.
+
+   ```
+   export BROWSERSTACK_USERNAME="my browserstack username"
+   export BROWSERSTACK_KEY="my browserstack access key"
+   ```
+
+   gulp test --browserstack >> prebid_test.log
+   
+   vim prebid_test.log
+
+
+2. Prepare Prebid Code
+
+   Update the package.json version to become the current release. Then commit your changes.
+
+   ```
+   git ci -m "Prebid 1.x.x Release"
+   git push
+   ```
+
+3. Publish Release
+
+   Make sure your Prebid code is clean and up to date
```

Comment:
What does this step mean? NPM? should include specifics.

Reply:
+1 on mkendall07's question

Reply:
Its just verifying that there are no more merges to master in between the process. Updated the step.